### PR TITLE
release 0.16.0

### LINKED ISSUE #249: Any plans for new release
Since the last release was in Aug is there any plan to do a new release?

### PR DESCRIPTION
**Pull request by jinankjain**
### Summary of the PR

New release for v0.16.0. Fixes #249 

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR are signed (with `git commit -s`), and the commit
  message has max 60 characters for the summary and max 75 characters for each
  description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Added
 
+## Changed
+
+# v0.16.0
+
+## Added
```
Comment (andreeaflorescu) [inline CHANGELOG.md:9]: The following need to be part of the `Added` section instead:
- Add support for coalesced MMIO (KVM_CAP_COALESCED_MMIO) from PR #244
- Add support for userspace NMI injection (KVM_NMI / KVM_CAP_USER_NMI) from #246
- Add support for userspace MSR handling #241 
- Add support for KVM_CAP_X86_SMM / KVM_SMI #242 

Can you please move them from `Changed`?

**Comment by roypat** [comment]
Should we maybe wait until https://github.com/rust-vmm/kvm-ioctls/pull/245 is merged, too? cc @00xc

**Comment by 00xc** [comment]
> Should we maybe wait until #245 is merged, too? cc @00xc

I won't have time for at least a few days to address the reviews in #245, so maybe go ahead with this release

**Comment by roypat** [comment]
Mh, either way, we might want to get https://github.com/rust-vmm/kvm-ioctls/pull/251 in, otherwise we'll just have to do another release to avoid `cargo audit` from firing

**Review by roypat — APPROVED** [review]

**Review by JonathanWoollett-Light — APPROVED** [review]

**Pull request merged**